packages/client/src/shared/ui/Section/index.tsx

```diff
@@ -1,14 +1,28 @@
 import classNames from 'classnames'
-import { PropsWithChildren } from 'react'
+import { CSSProperties, PropsWithChildren } from 'react'
 import s from './style.module.scss'
 
 type SectionProps = PropsWithChildren<{
   orientation?: 'column' | 'row'
+  pb?: boolean
+  alignItems?: 'stretch' | 'flex-start' | 'center' | 'flex-end'
 }>
 
-const Section = ({ children, orientation = 'column' }: SectionProps) => {
+const Section = ({
+  children,
+  orientation = 'column',
+  pb,
+  alignItems = 'stretch',
+}: SectionProps) => {
+  const isRow = orientation === 'row'
+  const alignmentStyle: CSSProperties = {
+    alignItems,
+  }
+
   return (
-    <section className={classNames(s.section, orientation === 'row' && s.row)}>
+    <section
+      className={classNames(s.section, isRow && s.row, pb && s.pb)}
```

Comment:
```suggestion
      className={classNames(s.section, { [s.row]: isRow, [s.pb]: pb })}
```

ты не используешь прелести classnames